rpcs3/Emu/SysCalls/Modules/sysPrxForUser.cpp

```diff
@@ -145,9 +145,13 @@ s32 console_putc()
 	throw EXCEPTION("");
 }
 
-s32 console_write()
+void console_write(vm::ptr<char> data, u32 len)
 {
-	throw EXCEPTION("");
+	// TODO: Move this to lv2 and just wrap it here.
```

Comment:
According to https://github.com/RPCS3/rpcs3/blob/master/rpcs3/Emu/SysCalls/SysCalls.cpp#L388, it is.